src/poly/dcrt/sampler/uniform.rs

```diff
@@ -60,39 +59,18 @@ impl PolyUniformSampler for DCRTPolyUniformSampler {
         ncol: usize,
         dist: DistType,
     ) -> Self::M {
-        #[cfg(feature = "disk")]
-        {
-            let mut new_matrix = DCRTPolyMatrix::new_empty(params, nrow, ncol);
-            let f = |row_offsets: Range<usize>, col_offsets: Range<usize>| -> Vec<Vec<DCRTPoly>> {
-                parallel_iter!(row_offsets)
-                    .map(|_| {
-                        parallel_iter!(col_offsets.clone())
-                            .map(|_| self.sample_poly(params, &dist))
-                            .collect()
-                    })
-                    .collect()
-            };
-            new_matrix.replace_entries(0..nrow, 0..ncol, f);
-            new_matrix
-        }
-        #[cfg(feature = "memory")]
-        {
-            let c: Vec<Vec<DCRTPoly>> = parallel_iter!(0..nrow)
+        let mut new_matrix = DCRTPolyMatrix::new_empty(params, nrow, ncol);
```

Comment:
i kept disk and memory seperate because memory case ::new_empty initialize with zero elements, not totally empty